jekyll/_cci2_ja/deploy-android-applications.adoc

```diff
@@ -0,0 +1,326 @@
+```
+
+NOTE: Executor にはデフォルト値がないため、設定ファイル内で設定する必要があります。
+
+[#run-each-command-individually]
+=== ii. 各コマンドをそれぞれ実行する
+
+ワークフロー内で各コマンドをそれぞれ実行する場合、以下のコマンドを追加する必要があります。
+
+- `decode-keystore`
+- `create-keystore-properties`
+- `create-google-play-key`
+- `fastlane-deploy`
+
+更に、Node Orb を使って `npm install` または `yarn install` のいずれかを実行する必要があります。
+
+deploy-to-play-store の方法では、このチュートリアルで先述したように環境変数を作成した場合、 `base64-keystore` 、 `release-key-alias` 、 `release-key-password` 、 `release-store-password` 、  `google-play-key` の各パラメーターの設定は不要です。
+
+以下は、この方法の場合の設定例です。
+
+```yaml
+orbs:
+  android: circleci/android@3.0.0
+  node: circleci/node@5.0.2
+jobs:
+  test-fastlane:
+      docker:
+        - image: cimg/android:2022.07
+      resource_class: large
+      steps:
+        - checkout
+        - node/install:
+            install-yarn: false
+            node-version: "16.13.0"
+        - run: npm install
+        - android/decode-keystore:
+            keystore-location: android/app/keystore
+        - android/create-keystore-properties:
+            working-directory: android
+        - android/create-google-play-key:
+            working-directory: android
+        - android/fastlane-deploy:
+            working-directory: android
+            lane-name: internal
+```
+
+[#set-up-config-without-the-android-orb]
+== 4b. Android Orb を使わずに設定をセットアップする
+
+. 以下のコマンドを `.circleci/config.yml` のデプロイジョブに追加し、先述の Base64 環境変数セット (`$BASE64_KEYSTORE`) のキーストアの暗号化を解除します。
++
+```yaml
+run:
+  name: Decode Android key store
+  command: echo $BASE64_KEYSTORE | base64 -d | tee keystore android/app/keystore > /dev/null
+```
+. 次に、Google Play ストアに作業をパブリッシュするために `keystore.properties` ファイルを生成します。
++
+これを行うためには、暗号化を解除したキーストアの場所を参照する `$RELEASE_KEYSTORE` という名前の別の環境変数を作成する必要があります。
++
+デプロイジョブに以下のコマンドを追加します。
++
+```yaml
+run:
+  name: Create keystore.properties
+  command: cd android && printf 'releaseKeyAlias=%s\nreleaseKeyPassword=%s\nreleaseKeyStore=%s\nreleaseStorePassword=%s' \
+  $RELEASE_KEY_ALIAS $RELEASE_KEY_PASSWORD $RELEASE_KEYSTORE $RELEASE_STORE_PASSWORD > keystore.properties
+```
+. 最後に、`api.json` から Google Play API キーを作成します。この値は環境変数 `$GOOGLE_PLAY_KEY` を使って保存済みなため、リモートリポジトリにアップロードするのではなく、実行時にこの値を参照して api.json ファイルを記述することができます。
++
+ジョブに以下のコマンドを追加します。
++
+```yaml
+run:
+  name: Create Google Play key
+  command: echo $GOOGLE_PLAY_KEY > google-play-key.json
+```
+. ここでアプリをデプロイするには、fastlane ステップを追加する必要があります。 これは `fastlane my_deployment_lane` を実行するコマンドを追加するだけで行えます。この場合、コマンドは以下のようになります。
++
+```yaml
+run: fastlane playstore
+```
++
+NOTE: パイプラインを実行しているイメージ上に fastlane をインストールする必要がある場合があります。 その場合、`sudo gem install fastlane` を実行します。
++
+fastlane はこのステップで Bundler を使用することを推奨しています。 Bundler の使用を選択した場合、Bundler のインストールステップを追加する必要があります。
++
+```
+run: sudo gem install fastlane
+```
++
+ `run: fastlane playstore` ステップを `run: bundle exec fastlane playstore` に置き換えます。
+
+[#next-steps]
+== 次のステップ
+
+- <<deploy-ios-applications#,iOS アプリのデプロイガイド>> も参照してください。
```

Comment:
the URL value is incorrect
https://circleci.com/docs/ja/deploy-ios-applications